stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -325,9 +331,16 @@ impl FromRow<NakamotoBlockHeader> for NakamotoBlockHeader {
         let parent_block_id = row.get("parent_block_id")?;
         let tx_merkle_root = row.get("tx_merkle_root")?;
         let state_index_root = row.get("state_index_root")?;
-        let signer_signature = row.get("signer_signature")?;
         let miner_signature = row.get("miner_signature")?;
         let signer_bitvec = row.get("signer_bitvec")?;
+        let signer_signature: SerdeValue = row.get_unwrap("signer_signature");
```

Comment:
Nit: as a convention, we use `serde_json::Value` instead of `SerdeValue` everywhere else in the codebase.

Reply:
Also, this shouldn't be `get_unwrap("signer_signature")`.  It should be `get("signer_signature")?"`